test/plugin/scenarios/webflux-scenario/support-version.list

```diff
@@ -23,3 +23,11 @@
 2.1.5.RELEASE
 2.1.6.RELEASE
 2.1.7.RELEASE
+2.1.8.RELEASE
+2.1.9.RELEASE
+2.1.10.RELEASE
+2.1.11.RELEASE
+2.1.12.RELEASE
+2.1.13.RELEASE
+2.1.14.RELEASE
+2.1.15.RELEASE
```

Comment:
Only pick up some of them to test, testing all versions requires many resources and makes the test slow

Reply:
ok,I will remove it